test/views/templates/VatOptOutSectionTemplateSpec.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.templates
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.twirl.api.HtmlFormat
+import views.ViewBaseSpec
+
+class  VatOptOutSectionTemplateSpec extends ViewBaseSpec {
+
+  "The vatOptOutSection" when {
+
+    object Selectors {
+      val vatOptOutHeading = "#vat-optout > h2:nth-child(1) a"
+      val vatOptOutParagraph = "#vat-optout > p:nth-child(2)"
+    }
+
+    def view: HtmlFormat.Appendable = views.html.templates.vatOptOutSection(Some(false))
+    implicit def document: Document = Jsoup.parse(view.body)
+
+    "the vat opt out feature" should {
```

Comment:
This `should` clause seems unnecessary. Might be worth just changing the `when` above to a `should` and then removing this extra clause.